data/card_catalogue/story-changing-golds.json

```diff
@@ -31,7 +31,7 @@
     "tier": "gold",
     "details": {
       "rules": {
-        "active": "Name a character not on the current map. They take 100 damage.",
+        "active": "Name a character not on the current map. They die instantly.",
```

Comment:
This is kind of bold but I think it'd be fun to see it play out. We can always lean on fake names to make this a little less powerful.